### PR TITLE
net-analyzer/nikto: version bump to 2.1.6_p20220602

### PR DESCRIPTION
**Pull request by CaseOf**

### CODE REVIEW

File: net-analyzer/nikto/nikto-2.1.6_p20220602.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	insinto /etc/nikto
+	newins nikto.conf.default nikto.conf
+
+	dobin nikto.pl replay.pl
+	dosym nikto.pl /usr/bin/nikto
+
+	insinto /usr/share/nikto
+	doins -r plugins templates databases
+
+	dodoc docs/*.txt
```
Comment (juippis) [inline net-analyzer/nikto/nikto-2.1.6_p20220602.ebuild:44]:
```
>>> Install net-analyzer/nikto-2.1.6_p20220602 into /var/tmp/portage/net-analyzer/nikto-2.1.6_p20220602/image
Traceback (most recent call last):
  File "/usr/lib/portage/python3.11/doins.py", line 607, in <module>
    sys.exit(main(sys.argv[1:]))
             ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 596, in main
    if _doins(
       ^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 439, in _doins
    return install_runner.install_file(source, os.path.dirname(dest))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 373, in install_file
    return self._ins_runner.run(source, dest_dir)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/portage/python3.11/doins.py", line 181, in run
    sstat = os.stat(source)
            ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: b'docs/*.txt'
 * ERROR: net-analyzer/nikto-2.1.6_p20220602::gentoo failed (install phase):
 *   dodoc failed
```
could you check what files you wanted to install here gets installed?

**Review by juippis — COMMENTED** [review]
Also we name the version when the commit was made, not when the ebuild was made. Because in theory you could bump the version any day, without commit changing. So if I looked correctly, the commit `dc98b86d3860ea18d82b5a427ab7abcd57e01552` was made .. 16th of May? so the correct version should be `2.1.6_p20220516`.

**Comment by CaseOf** [comment]
Hello, I did take the date of the merge of the pull request. Was it not the date taken for that commit? GitHub interface is very confusing.

**Comment by juippis** [comment]
Ah indeed, I browsed straight to  
https://github.com/sullo/nikto/commit/dc98b86d3860ea18d82b5a427ab7abcd57e01552.patch
`Date: Mon, 16 May 2022 11:18:26 -0700` 
but it was committed on 20220602. So the version is fine!

**Comment by CaseOf** [comment]
I did fix the doc issue in the ebuild.

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**